File: app/src/main/java/com/zy/ppmusic/utils/StreamUtils.java
package com.zy.ppmusic.utils;

import java.io.Closeable;
import java.io.IOException;

/**
 * @author ZhiTouPC
 */
public class StreamUtils {
    protected static void closeIo(Closeable... closeable){
        for (Closeable item : closeable) {
            if (item != null) {
                try {
                    item.close();
                } catch (IOException e) {
                    e.printStackTrace();
                }
            }
        }
    }
}

File: app/src/main/java/com/zy/ppmusic/utils/StreamUtils.kt
package com.zy.ppmusic.utils

import java.io.Closeable
import java.io.IOException

/**
 * @author stealfeam
 */
object StreamUtils {
    @JvmStatic
    fun closeIo(vararg closeable: Closeable?) {
        for (item in closeable) {
            if (item != null) {
                try {
                    item.close()
                } catch (e: IOException) {
                    e.printStackTrace()
                }

            }
        }
    }
}